tests/account/test_api.py

```diff
@@ -62,7 +63,9 @@ async def test_get(snapshot, spawn_client, static_time):
 async def test_edit(body, status, snapshot, spawn_client, resp_is, static_time):
     client = await spawn_client(authorize=True)
 
-    client.app["settings"].minimum_password_length = 8
+    await client.db.settings.find_one_and_update(
+        {"_id": "settings"}, {"$set": {"minimum_password_length": 8}}
+    )
```

Comment:
Should use the data layer method here:
```py
await get_data_from_app(client.app).settings.update(UpdateSettingsSchema(minimum_password_length=8))
```
Make this change everywhere else in this PR as necessary.